ui-v2/app/styles/routes/dc/services/index.scss

```diff
@@ -1,8 +1,9 @@
 // Services - Linked Services tab
-.gateway-services-list > ul {
-  @extend %gateway-services-list;
+// TODO - move this into composite-row
+.consul-gateway-services-list > ul {
+  @extend %consul-gateway-services-list;
 }
-%gateway-services-list > li:not(:first-child) {
+%consul-gateway-services-list > li:not(:first-child) {
```

Comment:
I much prefer the additional `consul-` here for the moment at least 👏 . Again not right now, but I'm pretty sure all of this can be condensed down into `%composite-row`.